pkg/controller/backup_controller.go

```diff
@@ -273,7 +274,11 @@ func (c *backupController) prepareBackupRequest(backup *api.Backup) *pkgbackup.R
 
 	// validate the storage location, and store the BackupStorageLocation API obj on the request
 	if storageLocation, err := c.backupLocationLister.BackupStorageLocations(request.Namespace).Get(request.Spec.StorageLocation); err != nil {
-		request.Status.ValidationErrors = append(request.Status.ValidationErrors, fmt.Sprintf("Error getting backup storage location: %v", err))
+		if strings.Contains(err.Error(), "not found") {
+			request.Status.ValidationErrors = append(request.Status.ValidationErrors, fmt.Sprintf("a BackupStorageLocation CRD needs to be deployed before this backup can be created. Error: %v", err))
```

Comment:
+1 to this message.

Reply:
I'd tweak this to clarify that a BSL *with the name specified in the backup spec* needs to be created/deployed, not just any BSL (similarly to the VSL messages below).  And maybe "before this backup can be executed" rather than "created".